utils/scurveAlgos.py

```diff
@@ -1075,6 +1095,7 @@ def fill2DScurveSummaryPlots(scurveTree, vfatHistos, vfatChanLUT, vfatHistosPanP
         # Fill Summary Histogram 
         if lutType is mappingNames[1] and vfatHistosPanPin2 is not None:
             if (stripPinOrChan < 64):
+                #### Off by one error...?
                 vfatHistos[event.vfatN].SetBinContent(63-(stripPinOrChan+1),chargeBin,event.Nhits)
                 vfatHistos[event.vfatN].SetBinError(63-(stripPinOrChan+1),chargeBin,sqrt(event.Nhits))
```

Comment:
This seems like an off by one error, namely the (64-1) and (stripPinOrChan+1) seem to be in opposition, eg 
```stripPinOrChan = 0   =>   bin = 62 ```
```stripPinOrChan = 63   =>   bin = -1```
Want to confirm before I try and change this

Reply:
I don't know...
* What is the valid range of values for `stripPinOrChan`?
* What do the current histograms look like (especially if you show the over/underflow stats)
* All things being equal, it looks to me like the `+1` should be outside the parentheses, and maybe also `vfatHistos` should be `vfatHistosPanPin2`...

Reply:
I did some tests, and having the +1 outside was correct. stripPinOrChan in {0,...,127} and the code works by if done by PanPin, the channels are split between the two histograms, `vfatHistos` and `vfatHistosPanPin2', each having a domain of [0,63].

All that being said, I made the fix!